packages/platform/platform-fastify/readme.md

```
@@ -0,0 +1,67 @@
+
+Run npm command (or yarn):
+
+```bash
+npm install --save @tsed/platform-fastify fastify
+```
+
+## Contributors
+
+Please read [contributing guidelines here](https://tsed.io/CONTRIBUTING.html)
+
+<a href="https://github.com/tsedio/ts-express-decorators/graphs/contributors"><img src="https://opencollective.com/tsed/contributors.svg?width=890" /></a>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: a

(MD033, no-inline-html)